gulpfile.js

```diff
@@ -24,6 +26,8 @@ gulp.task('minify', function () {
         output: {
           ascii_only: true
         }
+      }).on('error', function(e) {
```

Comment:
While debugging, the errors where not shown with file name and line number. Added error for easier debugging. Let me know if not needed.